docs/reference/flavors/index.md

```diff
@@ -48,6 +48,10 @@ general-purpose compute instance with 4 cores and 32 GiB RAM.
 * `g`: Virtual GPU. Instances launched with matching flavors have
   access to a
   [GPU](https://en.wikipedia.org/wiki/Graphics_processing_unit).
+* `c`: Physical CPUs. Instances launched with matching flavors will be provided
+  a physical CPU cores rather then virtual ones. These flavors are recommended
```

Comment:
```suggestion
* `c`: Physical CPUs. Instances launched with matching flavors are allocated
  physical CPU cores, rather than virtual ones. These flavors are recommended
```